src/index.rs

```diff
@@ -904,4 +904,23 @@ mod tests {
         assert!(new2 > old2);
         assert!(new2 > new1);
     }
+
```

Comment:
Idea for a test case:
1. Build the index of folder with paths e.g. `root/A` and `root/B`.
2. Create file by path `root/C`.
3. Call `update_one("root/C", id)`.
Two cases:
a) `id` matches the content of new resource
b) `id` does not 
4. What should be the result?

Reply:
Nice, alright. Should it replace the current one or is it a new unittest?

Reply:
New one